### DISCUSSION TITLE
How to ignore build of a branch?

### DISCUSSION THREAD

**Discussion by AnonymousXC** [vercel/community#1339] [Help] [answered]
Hi,
I want to ignore the build and deployment of a brach. Like, I have a `dev` branch and whenever I push in this branch. Vercel builds it and deploys it. I don't want to deploy it or build. I only want to build and deploy when I push to `main` branch.
Thanks

**Comment by amyegan** [comment, marked as answer]
You can [use Ignored Build Step](https://vercel.com/guides/how-do-i-use-the-ignored-build-step-field-on-vercel) to skip builds and deployments for all branches other than `main`. It would look something like 
```
if [ "$VERCEL_GIT_COMMIT_REF" == "main" ]; then exit 1; else exit 0; fi
```
